Add `align-self` utility classes (#2010)
shoelace-style/webawesome: packages/webawesome/docs/docs/utilities/align-items.md

---
title: Align Items

> description: Align items and align self utilities align items within flex and grid containers on the cross axis.

Reviewer: ```suggestion
description: Control cross-axis alignment in flex and grid containers with align-items and align-self utility classes.
```

The change here felt a bit awkward. Here's an attempt to rewrite it.

    background-color: var(--layout-example-element-background);
    border-radius: var(--wa-border-radius-s);
    min-block-size: 1em;

>     &.de-emphasize {

Reviewer: `de-lightful` idea! If we use this elsewhere, can we abstract it out (present us or future us)?

Author: Promoted in https://github.com/shoelace-style/webawesome/pull/2010/changes/7aecd291da50fc7fb571b05654a86e1e0f484231!

Let me know if you feel there's a better place or scope for this.
